### PR TITLE
fix(storefront): BCTHEME-909 Social media icons failing wave scan with Empty Link.

### PR DESCRIPTION
**Pull request by bc-yevhenii-buliuk**
#### What?
This PR removes the warning about empty links in WAVE (web accessibility evaluation tool) service.

#### Tickets / Documentation
[BCTHEME-909](https://jira.bigcommerce.com/browse/BCTHEME-909)

#### Screen Recording

WAVE scan with warning:

https://user-images.githubusercontent.com/82589781/163811745-272fa52d-6a1b-437c-be66-8a59e43110cb.mov

WAVE scan without warning:

https://user-images.githubusercontent.com/82589781/163811673-6fc9bb1f-2e38-42d6-b969-045639237924.mov

### CODE REVIEW

File: templates/components/common/social-links.html [hunk 1]
```diff
@@ -1,11 +1,25 @@
 {{#if social_media}}
     <ul class="socialLinks socialLinks--alt">
         {{#each social_media}}
-            <li class="socialLinks-item">
-                <a class="icon icon--{{display_name}}" href="{{url}}" target="_blank" rel="noopener">
-                    <svg><use xlink:href="#icon-{{display_name}}" /></svg>
-                </a>
-            </li>
+            {{#if display_name '===' 'facebook'}}
```
Comment (bc-alexsaiannyi) [inline templates/components/common/social-links.html:4]: do we know how many `display_name` options are available at all?

Comment (bc-yevhenii-buliuk) [reply templates/components/common/social-links.html:4]: @bc-alexsaiannyi I took all possible options (at the moment) from the Control Panel settings.

Comment (bc-alexsaiannyi) [reply templates/components/common/social-links.html:4]: I don't see that specific `display_name` has any influence on showing `components/common/social-links-item` component or not. It seems as the reason to show social-links-item component depends on `social_media` condition on higher level. So if we remove checking  `display_name`  will this affect us at all?

Comment (bc-yevhenii-buliuk) [reply templates/components/common/social-links.html]: @bc-alexsaiannyi thanks, I changed my previous approach to solve this problem.

**Review by bc-alexsaiannyi — APPROVED** [review]

**Review by yurytut1993 — APPROVED** [review]

**Review by bc-vlad-dlogush — APPROVED** [review]

**Review by bc-azvierieva — APPROVED** [review]

**Pull request merged**